packages/ketchup/src/components/kup-activity-timeline/kup-activity-timeline.scss

```diff
@@ -0,0 +1,163 @@
+.atm-container {
```

Comment:
Remove the cabled value from the SCSS.
Use the token that already exists and are used around the application. 
An example is somethink like that : 
--kup_box_background_color: var(--kup-box-background-color, var(--kup-layer-1));
--kup_box_grid_gap: var(--kup-box-grid-gap, var(--kup-space-03));

These are tokens ( like kup-layer-1 or kup-space-03) that are already prepared so that you don't have to decide/rewrite the value. Use these already existing tokens to speed up you work and get the results aligned with the other components inside ketchup. And use the variable (--kup_box_grid_gap etc.. ) where the first value is the internal variable and the second value ( --kup-box-grid-gap ) to manage the SCSS 

Example : 
--kup_atm_container_padding: var(--kup-atm-container-pading, var(--kup-space-03));
.atm-container .atm-event .atm-content {
  padding: var(--kup_atm_container_padding);
}